manifests/k/Kaspersky/Lab/1.0/Kaspersky.Lab.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Kaspersky.Lab
+PackageVersion: 1.0
+Installers:
+- Architecture: x64
+  InstallerType: portable
+  InstallerUrl: https://download1506.mediafire.com/wad68t3os59g/xsgspau91udkrkw/Kaspersky_Endpoint_Security_11.6_com_Agent.exe

```

Comment:
A mediafire link isn't suitable for an InstallerUrl. Where did you get this from?